### PR TITLE
community: Add Support for Loading Documents from Alibaba Cloud OSS file.

### PR DESCRIPTION
**Pull request by pcliupc**
- **Description:** Add document loader for loading file from alibaba cloud OSS (object storage service)
- **Issue:** None
- **Dependencies:** Install oss2 python package.
- **Twitter handle:** None

### CODE REVIEW

**Comment by pcliupc** [comment]
@baskaryan Please help review this pr.

File: libs/community/langchain_community/document_loaders/oss_file.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import tempfile
+from typing import Any, List, Optional
+
+from langchain_core.documents import Document
+
+from langchain_community.document_loaders.base import BaseLoader
+from langchain_community.document_loaders.unstructured import UnstructuredFileLoader
+
+
+class OSSFileLoader(BaseLoader):
```
Comment (baskaryan) [inline libs/community/langchain_community/document_loaders/oss_file.py:13]: lets call this AlibabaOSSFileLoader (and the file alibaba_oss_file_loader.py)

Comment (pcliupc) [reply libs/community/langchain_community/document_loaders/oss_file.py]: fixed

**Comment by eyurtsev** [comment]
I'd suggest following this PR: https://github.com/langchain-ai/langchain/pull/21957

1) We generally just need a primitive which is a BaseBlobLoader that only yields Blobs, without any parsing logic at all.
2) Could you remove any changes in langchain package. Changes should be strictly in langchain-community

**Comment by pcliupc** [comment]
> I'd suggest following this PR: #21957
> 
> 1. We generally just need a primitive which is a BaseBlobLoader that only yields Blobs, without any parsing logic at all.
> 2. Could you remove any changes in langchain package. Changes should be strictly in langchain-community

sure, I will keep an eye on this PR.

**Comment by pcliupc** [comment]
> I'd suggest following this PR: #21957
> 
> 1. We generally just need a primitive which is a BaseBlobLoader that only yields Blobs, without any parsing logic at all.
> 2. Could you remove any changes in langchain package. Changes should be strictly in langchain-community
@eyurtsev The CloudBlobLoader implemented in this PR #21957 supports three kinds of cloud path, are you suggesting extending CloudBlobLoader to support more? And BTW, CloudBlobLoader relies on [CloudPathLib](https://cloudpathlib.drivendata.org/stable/) library which also only supports three kinds of cloud path.

**Pull request closed without merging**